apps/nextjs-website/src/_contents/products.ts

```diff
@@ -4,12 +4,12 @@ import { pagoPaOverview } from '@/_contents/pagoPa/overview';
 import { pagoPaGuides } from '@/_contents/pagoPa/guides';
 import { appIO } from './appIo/appIO';
 import { ioSign } from './ioSign/ioSign';
-import { pagoPA } from './pagoPa/pagoPA';
+import { pagoPa } from './pagoPa/pagoPa';
 
 export const overviews = [appIoOverview, ioSignOverview, pagoPaOverview];
 export const quickStarts = [];
 export const apis = [];
 export const tutorials = [];
 export const tools = [];
 export const productsGuides = [pagoPaGuides];
```

Comment:
The `productsGuides` is going to be ambiguous once the gitbook guides are integrated. What do you think to rename it to `guideHomepages`?